_component_design/file-tile.md

```diff
@@ -0,0 +1,316 @@
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+A file tile is part of a file handling workflow and is one approach you can use to indicate that a file has been uploaded to a control panel page.
+
+Use a file tile when you want to make the file more visually prominent. A file tile is also easier for a user to interact with.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a file tile:
+
+- Include an icon in the file tile to represent the file format. Only use icons available in the [Helix Toolkit](https://rackerlabs.github.io/helix-ui/components/icons/#available-icons).
+- Do not use branded icons.
+- Consider where you place file tiles in a view. Avoid making the user scroll down a long list of file tiles to see important information.
+- Only the user who uploaded the file can remove it. A file tile should be read-only for all other users.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Specifications
+
+Refer to the following specifications when you construct a file tile.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A file tile contains the following elements:
+
+- **Container (required)**: The container holds the file icon, file name, file size, and remove X. The file download process starts when the user clicks anywhere in the container *except the Remove X*.
+- **File icon (required)**: The file icon represents the file format.
+- **File name (required)**: The file name displays the name of the file.
+- **File size (required)**: The file size displays the size of the file.
+- **Remove X (optional)**: When the user clicks **X**, the file is removed from the page. Only the user who uploaded the file can remove it.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"File tile composition"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/content-areas/file-tiles/filetiles-composition.png" width="419"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Style
+
+Use these specifications when you construct a file tile.
+
```

Comment:
visual needs to change. the icon is called "times" so the style redline for the remove X should be icon: times

Reply:
This was resolved